server/express.js

```diff
@@ -181,10 +181,13 @@ function makeApp(authAddress, cdapConfig, uiSettings) {
             reportUri: `https://csp.withgoogle.com/csp/cdap`,
           },
         },
-        hsts: {
-          includeSubDomains: true,
-          preload: true,
-        },
+        strictTransportSecurity: cdapConfig["hsts.enabled"]
```

Comment:
The boolean value is already parsed, right?

Reply:
fixed